app/views/shared/_navbar.html.erb

```diff
@@ -1,17 +1,29 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon px-3 sticky-top">
   <%= link_to root_path do %>
-    <%= image_tag "logo_small.png", alt: "alttext", class: "logo" %>
+    <%= image_tag "logo_red.png", alt: "alttext", class: "logo" %>
   <% end %>
-  <%# <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+
+  <!-- <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
     <span class="navbar-toggler-icon"></span>
-  </button> %>
+  </button> -->
 
-  <div class="search_bar">
+  <div class="search_bar navbar-nav ml-auto mr-auto">
     <form class="form-inline input-group">
-      <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
-      <button class="btn btn-outline-danger my-2 my-sm-0" type="submit">Search</button>
+      <input class="form-control mr-sm-2" type="search" placeholder="Find a venue" aria-label="Search">
```

Comment:
I was wondering if the placeholder should be found by type of venue or by location.